src/components/BugReportDialog.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <Dialog :visible="visible" @update:visible="$emit('hide')">
+        <template #header>
+            <h3>Report a bug</h3>
+        </template>
+        <div class="flex flex-column pt-4">
+            <span class="p-float-label mb-5">
```

Comment:
Span shouldn't be used as a container for other elements. You can make there a normal div with styling. Even IDE is badly formatting these elements inside